src/controller/ticket.js

```diff
@@ -2,61 +2,61 @@ const ticketService = require("../service/ticket")
 const buses = require("../models/bus")
 const ticket = require("../models/ticket")
 
-const bookticket = async(req,res)=>{
+const bookTicket = async(req,res)=>{
     try {
         const ticketDetails = req.body
         const busNumber = ticketDetails.busNumber
         const availability = await buses.findOne({busNumber})
         if(availability.avaiableSeat.length>0 && ticketDetails.seatcount<=availability.avaiableSeat.length){
-           const ticket = await ticketService.bookticket(ticketDetails,availability.date,availability.avaiableSeat)
-           const update = await ticketService.updatebusticket(ticketDetails.seatcount,busNumber)
-           res.status(201).json({ticket:ticket,update:update,"msg":"ticket is successfully booked"})
+           const ticket = await ticketService.bookTicket(ticketDetails, availability.date, availability.avaiableSeat)
+           const update = await ticketService.updateBusTicket(ticketDetails.seatcount, busNumber)
+           res.status(201).json({ticket, update, message:"ticket is successfully booked"})
         }else{
-           res.json({"message":"seat are full"})
+           res.json({message:"seat are full"})
         }
     } catch (err) {
         console.log(err)
-        res.json({err:err,message:" oofhs! something wrong"})
+        res.json({err, message:" oops! something wrong"})
     }
     
 }
 
-const canacelTicket = async(req,res)=>{
+const cancelTicket = async(req,res)=>{
     try {
         const ticketDetails = req.body
         const email = ticketDetails.email
         const existingticket = await ticket.findOne({email}) || null
         const pnr = existingticket?.pnr 
```

Comment:
instead of pnr, descriptive word could be used

Reply:
pnr is a long word so only I used this. instead of pnr I used PNR.